### PR TITLE
random text

### PR DESCRIPTION
**Pull request by Pozik7**
### HTML
added messageContainer to the div. this structure ensures that the message appears below the button

### CSS
flex-direction: column added to stack the button and message vertically
margin-bottom is added to the button to separate it from the text.

### JS
Code changed to print random text from array
removed messageContainer and .container 

image

```

### CODE REVIEW

**Review by VicDosya — CHANGES_REQUESTED** [review]
Nicely done!
Few changes remaining:
Make sure the button is ABOVE the printed text.
Also make sure only the text content of ONE div changes (not in addition). as we talked in discord.

**Comment by Pozik7** [comment]
> Nicely done! Few changes remaining: Make sure the button is ABOVE the printed text. Also make sure only the text content of ONE div changes (not in addition). as we talked in discord.

Updated please make sure this is correct.

File: script.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+  "I have completed the task as per your instructions. May we now proceed?",
+  "Task completed successfully. What would you like to do next?",
+  "Your request has been fulfilled. Shall we continue?",
+  "The task is done. What is the next step?",
+  "Mission accomplished. How should we proceed from here?",
+];
+
+button.addEventListener("click", function () {
+  const messageContainer = document.getElementById("messageContainer");
+  const randomIndex = Math.floor(Date.now() % messages.length);
```
Comment (VicDosya) [inline script.js]: Why `Date.now() `?
Let's go with a simple approach:
Math.floor(Math.random() * messages.length);

**Review by VicDosya — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (Pozik7) [reply script.js]: Done

**Review by VicDosya — APPROVED** [review]
Nice!

**Pull request merged**